db/migrate/20170105165853_add_timestamps_to_bus_stops.rb

```diff
@@ -0,0 +1,9 @@
+class AddTimestampsToBusStops < ActiveRecord::Migration
+  def change
+    change_table :bus_stops do |t|
+      t.timestamps null: false
```

Comment:
Does this cause any problems with existing data?  That is, does AR fill in the `created_at` with... something if its missing?